docker-compose.yml

```
@@ -213,20 +211,19 @@ services:
       - CONNECTOR_AUTO=true # Enable/disable auto-import of file
       - CONNECTOR_ONLY_CONTEXTUAL=false # Only extract data related to an entity (a report, a threat actor, etc.)
       - CONNECTOR_CONFIDENCE_LEVEL=15 # From 0 (Unknown) to 100 (Fully trusted)
-      - OPENCTI_JSON_LOGGING=true # Enable / disable JSON logging
-      - CONNECTOR_LOG_LEVEL=info=error
+      - CONNECTOR_LOG_LEVEL=debug
```

```suggestion
      - CONNECTOR_LOG_LEVEL=error
```
I assume this was also changed for debugging?

---

Yes, it was changed for debugging purposes.